index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="hero">
+      <header class="header-sec">
+        <h1 class="logo">Welcome</h1>
+        <nav class="navlinks">
+          <ul class="links">
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </nav>
+        <div class="toggleMenu">
+          <div class="menu"></div>
+        </div>
+      </header>
+
+      <section class="hero-intro">
+        <div class="title">
+          <h1>
+            Hey there. I’m Evans. <br />
+            <span class="inner-h1"> I’m a software developer</span>
+          </h1>
+        </div>
+        <div class="description">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials">
+          <a href=""><i class="fa-brands fa-github"></i></a>
+          <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+          <a href=""><img src="/assets/Vector.png" alt="" /></a>
+          <a href=""><i class="fa-brands fa-twitter"></i></a>
+          <a href=""><i class="fa-solid fa-m"></i></a>
+        </div>
```

Comment:
- Great Job implementing the setup of your mobile portfolio 👍🏽 

- I think it will be better if you replaced the list of icons using a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (a tag) instead of a `<div>`. 
We can replace the `<div class="socials">` with `<ul class="socials">` . 🤗